session/bootstrap.go

```diff
@@ -1125,6 +1133,72 @@ func insertBuiltinBindInfoRow(s Session) {
 	mustExecute(s, sql)
 }
 
+func upgradeToVer51(s Session, ver int64) {
```

Comment:
This function may cause data inconsistency. One user upgraded the cluster from 3.0.8 to 4.0.12, met the bind_info table inconsistent, and upgrades from 4.0.12 to 5.4.1, met the inconsistent error again. I think it is due to upgradeToVer51 changed to upgradeToVer67 in 5.4.1.